### ISSUE TITLE
flac-seektable does not build on node 12

### ISSUE THREAD

**Issue by josephg** [lammas/flac-seektable#1]
The deprecation warnings under node 11 are now errors:

```
sephsmac:ff josephg$ node --version
v12.1.0
sephsmac:ff josephg$ npm i flac-seektable

> flac-seektable@0.1.3 install /Users/josephg/test/ff/node_modules/flac-seektable
> node-gyp rebuild

  CXX(target) Release/obj.target/flac-seektable/src/main.o
../src/main.cpp:303:37: error: no matching member function for call to
      'ToObject'
        Local<Object> bufferObj = info[0]->ToObject();
                                  ~~~~~~~~~^~~~~~~~
/Users/josephg/.node-gyp/12.1.0/include/node/v8.h:2532:44: note: candidate
      function not viable: requires single argument 'context', but no arguments
      were provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Object> ToObject(
                                           ^
/Users/josephg/.node-gyp/12.1.0/include/node/v8.h:2546:35: note: candidate
      function not viable: requires single argument 'isolate', but no arguments
      were provided
                    Local<Object> ToObject(Isolate* isolate) const);
                                  ^
../src/main.cpp:381:10: warning: 'Set' is deprecated: Use maybe version
      [-Wdeprecated-declarations]
                        tags->Set(index, New(*it).ToLocalChecked());
                              ^
/Users/josephg/.node-gyp/12.1.0/include/node/v8.h:3367:3: note: 'Set' has been
      explicitly marked deprecated here
  V8_DEPRECATE_SOON("Use maybe version",
  ^
/Users/josephg/.node-gyp/12.1.0/include/node/v8config.h:322:29: note: expanded
      from macro 'V8_DEPRECATE_SOON'
  declarator __attribute__((deprecated(message)))
                            ^
../src/main.cpp:392:15: warning: 'Set' is deprecated: Use maybe version
      [-Wdeprecated-declarations]
                seekpoints->Set(i, pt);
                            ^
/Users/josephg/.node-gyp/12.1.0/include/node/v8.h:3367:3: note: 'Set' has been
      explicitly marked deprecated here
  V8_DEPRECATE_SOON("Use maybe version",
  ^
/Users/josephg/.node-gyp/12.1.0/include/node/v8config.h:322:29: note: expanded
      from macro 'V8_DEPRECATE_SOON'
  declarator __attribute__((deprecated(message)))
                            ^
2 warnings and 1 error generated.
make: *** [Release/obj.target/flac-seektable/src/main.o] Error 1
```

nan is updated & works for node 12, though just like my module I'm not sure if updating it here will fix the issue. Also [napi](https://nodejs.org/api/n-api.html) works great, and has been backported to earlier versions of node.

**Comment by lammas** [comment]
It seems v8 has introduced quite a few breaking changes. I can currently push a version that fixes it for node v12, but a lot of the APIs will be deprecated soon. Hopefully I will have enough time during the summer to address these.

**Comment by lammas** [comment]
Published v0.1.4 - it should be working with v12 now.
